website/source/docs/commands/verify.html.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: "docs"
+page_title: "Command: verify"
+sidebar_current: "docs-commands-verify"
+description: |-
+  The `terraform verify` command is used to validate the format and structure of the terraform files.
+---
+
+# Command: verify
+
+The `terraform verify` command is used to validate the format and structure of the terraform files.
+Terraform performas an analysis of the terraform files, and will display information about missing
+ variables and errors in the structure of the files.
```

Comment:
Can we reword this a little bit to not mention `format` and `structure`? What we're really checking is _syntax_ mostly.

Formatting will be checked by `fmt` per https://github.com/hashicorp/terraform/pull/4955 eventually.

I would in fact even explicitly mentioned in a separate paragraph that this command does not check the format of your code. We may expand that paragraph later on when we merge `fmt` as linked above.

------------

It's pretty much like `go vet` VS `gofmt` whereas `go vet` is what we do.